Docs/source/developers/particles.rst

```diff
@@ -107,19 +107,49 @@ To reduce numerical artifacts at the boundary of a mesh-refinement patch, WarpX
 
    Buffers are complex!
 
-Runtime particle attributes
----------------------------
-
-WarpX allows extra runtime attributes to be added to particle containers (through ``AddRealComp("attrname")`` or ``AddIntComp("attrname")``). The attribute name can then be used to access the values of that attribute. For example, using a particle iterator, ``pti``, to loop over the particles the command ``pti.GetAttribs(particle_comps["attrname"]).dataPtr();`` will return the values of the ``"attrname"`` attribute.
-
-Commonly used runtime attributes are described in the table below.
+Particle attributes
+-------------------
 
-==================  ================  =================================  =============
+WarpX adds the following particle attributes by default to WarpX particles.
+These attributes are either stored in an Array-of-Struct (AoS) or Struct-of-Array (SoA) location of the AMReX particle containers.
+The data structures for those are either pre-described at compile-time (CT) or runtime (RT).
+
+====================  ================  ==================================  ===== ==== =====================
+Attribute name        ``int``/``real``  Description                         Where When Notes
+====================  ================  ==================================  ===== ==== =====================
+``position_x/y/z``    ``real``          Particle position.                  AoS   CT
+``cpu``               ``int``           CPU index where the particle        AoS   CT
+                                        was created.
+``id``                ``int``           CPU-local particle index            AoS   CT
+                                        where the particle was created.
+``ionization_level``  ``int``           Ion ionization level                SoA   RT   Added when ionization
```

Comment:
Memo to myself for another time: this needs to be renamed in the code (x-ref: #1652)